subprojects/android-test/test-instrumentation-runner/src/main/kotlin/com/avito/android/runner/SystemDialogsManager.kt

```diff
@@ -27,11 +28,13 @@ class SystemDialogsManager(private val errorsReporter: ErrorsReporter) {
             errorsReporter.reportError(t)
         }
 
-        try {
-            dismissHiddenApiDialog()
-        } catch (t: Throwable) {
-            Log.v(TAG, "Failed to close hidden api dialog: ${t.message}")
-            errorsReporter.reportError(t)
+        if (Build.VERSION.SDK_INT >= Build.VERSION_CODES.P) {
+            try {
+                dismissHiddenApiDialog()
```

Comment:
Would it increase test flakiness because of dialogs?

Reply:
This dialog is about restricted API and must be only on 28+ API